### DISCUSSION TITLE
The size of the file increases after minifying with swc

### DISCUSSION THREAD

**Discussion by nixjs** [swc-project/swc#7962] [Q&A] [answered]
I don't understand why when I use swc to minify js, the size of the javascript file increases compared to the original size.
- The picture on the left is before compression:  `2.514 bytes`
- The image on the right is after compression: `3.333bytes`
<img width="527" alt="image" src="https://github.com/swc-project/swc/assets/23132483/96b7fb83-0bf0-4f30-b6af-8af72f329586">

**Comment by kdy1** [comment, marked as answer]
I think you used `transform()`. Use `minify()` instead

**Reply by nixjs** [reply]
Thank you. I resolved
